### PR TITLE
package.json - update node engines

### LINKED ISSUE #407: update node engines in package.json as per openid-client 5.1.1 requirements
openid-client 5.1.1 has `"node": "^12.19.0 || ^14.15.0 || ^16.13.0` node engine conditions to use specific node version so we should update at our side also.

discussion ref https://github.com/GluuFederation/gluu-passport/pull/397/files#r785360818

### PR DESCRIPTION
**Pull request by kdhttps**
close #407

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     ]
   },
   "engines": {
-    "node": ">=12.19.0 <=16.13.0"
+    "node": "^12.19.0 || ^14.15.0 || ^16.13.0"
```
Comment (christian-hawk) [inline package.json:13]: as https://github.com/GluuFederation/gluu-passport/pull/397/files#r785360818 I thought we were gonna move to 16 only. Why do we still need 12 and 14?

**Review by christian-hawk — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kdhttps) [reply package.json:13]: No mate, Our passport supports 14 and 16 both. I will remove 12.

Comment (kdhttps) [reply package.json]: Done! @christian-hawk  mate, do you also want me to update the github workflows?

**Comment by kdhttps** [comment]
run tests

**Comment by christian-hawk** [comment]
surpassed by #451

**Pull request closed without merging**